### PR TITLE
Add a new sample service for Endpoints Frameworks for Python

### PR DESCRIPTION
**Pull request by inklesspen**
The IATA sample service provides examples of CRUD operations, managing
a list of airports and their corresponding IATA codes. This service
will be used in documentation for the Endpoints Frameworks.

### CODE REVIEW

File: appengine/standard/endpoints-frameworks-v2/iata/app.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+runtime: python27
+threadsafe: true
+api_version: 1
+basic_scaling:
+  max_instances: 2
+
+#[START_EXCLUDE]
```
Comment (jonparrott) [inline appengine/standard/endpoints-frameworks-v2/iata/app.yaml]: This shouldn't be necessary.

File: appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from google.appengine.ext import vendor
```
Comment (jonparrott) [inline appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py]: needs a license header.

File: appengine/standard/endpoints-frameworks-v2/iata/app.yaml (hunk 1 above)
Comment (inklesspen) [reply appengine/standard/endpoints-frameworks-v2/iata/app.yaml]: These are in the app.yaml in appengine/standard/endpoints-frameworks-v2/echo; do you know why they're there? I just mimicked that.

File: appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py (hunk 2 above)
Comment (inklesspen) [reply appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py]: This file is identical to (and was copied from) the equivalent file in appengine/standard/endpoints-frameworks-v2/echo.

Comment (jonparrott) [reply appengine/standard/endpoints-frameworks-v2/iata/appengine_config.py]: Still needs a license header. :)

**Comment by inklesspen** [comment]
Done.

**Pull request merged**